Sources/Citadel/SFTP/SFTPMessage.swift

```diff
@@ -286,13 +320,26 @@ public enum SFTPMessage {
     /// No response, directory gets created or an error is thrown.
     case mkdir(MkDir)
     
+    case stat(Stat)
```

Comment:
Ideally some nice comments like the other enums :) (Since I don't immediately know what stat and lstat are or what attributes we're talking about here)